src/commands/deployContainerApp/deployContainerApp.ts

```diff
@@ -60,18 +61,28 @@ export async function deployContainerApp(context: IActionContext, node?: Contain
     wizardContext.telemetry.properties.revisionMode = item.containerApp.revisionsMode;
 
     const confirmationViewTitle: string = localize('summary', 'Summary');
-    const confirmationViewDescription: string = localize('viewDescription', 'Please select an input you would like to change. Note: Any input proceeding the changed input may need to change as well');
-    const confirmationViewTabTitle: string = localize('deployContainerAppTabTitle', 'Summary - Deploy Image to Container App');
-    const title: string = localize('deployContainerAppTitle', 'Deploy image to container app');
+    let confirmationViewDescription: string = localize('viewDescription', 'Please select an input you would like to change. Note: Any input proceeding the changed input may need to change as well');
```

Comment:
nit: Right now, it's not a may, it's guaranteed that it needs to change, right?

Also maybe we should note that a changed input may lead to different prompts?